bitnami/postgresql/values-production.yaml

```diff
@@ -82,9 +82,15 @@ volumePermissions:
 ## Pod Security Context
 ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 ##
-securityContext:
+podSecurityContext:
```

Comment:
This should still be `securityContext`, right?

Reply:
Yes it should! I've pushed a new commit which fixes it